tests/robot-tests/tests/libs/charts.robot

```diff
@@ -141,11 +141,26 @@ user checks chart tooltip item contains
     user waits until element is visible    ${element}
     user waits until element contains    ${element}    ${text}
 
+user checks map tooltip label contains
+    [Arguments]    ${locator}    ${text}
+    user waits until parent contains element    ${locator}    css:[data-testid="chartTooltip-label"]
+    ${element}=    get child element    ${locator}    css:[data-testid="chartTooltip-label"]
```

Comment:
In cases where we use testids, we should be able to use our custom selector for it i.e: `testid:chartTooltip-label`

We might want to do this for the other testid selectors further down as well.

Reply:
Done